### PR TITLE
fix(router-store): also handle routerNavigated action in reducer

### PR DESCRIPTION
**Pull request by dhhyi**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/ngrx/platform/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Documentation has been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

```
[x] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?

I wanted to use a Resolver to set a data field on a lazy-loaded route and then later on select the data (which we are mapping with a custom router serializer) from the store. I could trace that the custom serializer was called correctly two times (routerNavigation and routerNavigated actions), but the second serialization was not triggering a store modification. And only the second serialization was containing the data field I had on the lazy-loaded route.

tl;dr: the routerNavigated action has the serialized routerState as payload, but it is not persisted in the store.

## What is the new behavior?

routerNavigated actions are handled by the routerReducer.

## Does this PR introduce a breaking change?

```
[ ] Yes
[x] No?
```

## Other information

If you want, I can try to provide a minimal error example. Please tell.

### CODE REVIEW

**Comment by brandonroberts** [comment]
Thanks @dhhyi. This is already handled by adjusting the navigation action timing documented here. https://ngrx.io/guide/router-store/configuration#navigation-action-timing

**Comment by dhhyi** [comment]
Hi @brandonroberts,
I was afraid, you were going to point me that way. If we switch to PostActivation timing, something else breaks, because all navigation actions arrive later. You know, enterprise software...

Shouldn't this also work for PreActivation timing?
